courses/machine_learning/deepdive/05_review/5_train.ipynb

```diff
@@ -19,19 +19,20 @@
   },
```

Comment:
<p>same here. Cloud ML Engine -&gt; Cloiud AI Platform in each instance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/730/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done!</p>